alibi_detect/cd/pytorch/classifier.py

```diff
@@ -149,9 +145,24 @@ def score(self, x: np.ndarray) -> float:
             train_args = [model, nn.CrossEntropyLoss(), dl_tr, self.device]
             trainer(*train_args, **self.train_kwargs)  # type: ignore
             preds = predict_batch(x_te, model.eval(), device=self.device, batch_size=self.dl_kwargs['batch_size'])
-            preds_oof.append(preds)
-            idx_oof.append(idx_te)
-        preds_oof = np.concatenate(preds_oof, axis=0).argmax(axis=1)
-        idx_oof = np.concatenate(idx_oof, axis=0)
-        drift_metric = self.metric_fn(y[idx_oof], preds_oof)
-        return drift_metric
+            preds_oof_list.append(preds)
+            idx_oof_list.append(idx_te)
+        preds_oof = np.concatenate(preds_oof_list, axis=0).argmax(axis=1)
+        idx_oof = np.concatenate(idx_oof_list, axis=0)
+
```

Comment:
From here on it looks like the functionality is framework-agnostic and the same as e.g. the TensorFlow one. Could be a good idea to add this as a function of the base method?